epics-team-builder/src/widgets/SetupPage/components/ManuallyAssignProjects.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { Card, Table, Button, CardDeck } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+export default class ManuallyAssignProjects extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projectData: [{}],
+      studentData: [{}]
+    };
+  }
+
+  handleChangeProject(e) {
+    const id = e.target.id;
+
+    if (document.getElementById(id).checked) {
+      console.log('ID: ' + id);
+    }
+  }
+
+  handleChange = e => {
+    const id = e.target.id;
+    const value = e.target.value;
+    /* console.log(e.target.id) ;
+    console.log(document.getElementById(id).checked) ; */
+  };
+
+  handleClickStudent(e) {
+    const id = e.target.id;
+    var checkedStatus = document.getElementById(id).checked;
+    console.log(document.getElementById(id).checked);
+    /*  console.log(tempstudentIdArray[1]) ; */
+  }
+
+  addProjectToStudent(projects, students) {
+    var status = false;
+    var currentProjectName = '';
+    var checkedStudents = [];
+    //var mappedStudents = {};
+    var mappedStudents = {
+      projects: [{ projectID: '', studentIDs: [] }]
+    };
+
+    for (var j = 0; j < projects.length; j++) {
+      var projectName = projects[j]['name'];
+
+      if (document.getElementById(projectName).checked) {
+        currentProjectName = projectName;
+        mappedStudents.projects[j] = {};
+        mappedStudents.projects[j]["projectID"] = currentProjectName;
+        console.log('current project name is' + currentProjectName);
+
+        for (var i = 0; i < students.length; i++) {
+          var currStudentID = students[i]['id'];
+
+          if (document.getElementById(currStudentID).checked) {
+            console.log(currStudentID + ' Student is checked');
+            checkedStudents.push(currStudentID);
+          }
+        }
+
+        for (var k = 0; k < checkedStudents.length; k++) {
+          mappedStudents.projects[j][k] = '';
+          mappedStudents.projects[j][k] = checkedStudents[k];
+        }
+
+        console.log(mappedStudents);
+        this.props.assignProjToStud(mappedStudents);
+      }
+    }
+
```

Comment:
Once the students have been assigned, we also need to stop displaying those students in the table somehow, try to figure out how to implement that here as well.